### PR TITLE
feat(emeis-options): custom user status component

### PR DESCRIPTION
**Pull request by derrabauke**
This adds a custom status component in the user detail view. It will be inline with the existing `inactive` badge on top of the form.

### CODE REVIEW

File: tests/dummy/app/components/dummy-status/dummy-status.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import Component from "@glimmer/component";
+
+export default class DummyStatusComponent extends Component {
+  get stati() {
```
Comment (czosel) [inline tests/dummy/app/components/dummy-status/dummy-status.js:4]: It seems that in english it's called `statuses` (https://english.stackexchange.com/questions/877/what-is-the-plural-form-of-status)
```suggestion
  get statuses() {
```

Comment (czosel) [reply tests/dummy/app/components/dummy-status/dummy-status.js:4]: Ah sorry I just realized this is only testing code. Feel free to leave it.

**Review by czosel — APPROVED** [review]
:+1:

File: tests/dummy/app/components/dummy-status/dummy-status.js (hunk 1 above)
Comment (derrabauke) [reply tests/dummy/app/components/dummy-status/dummy-status.js]: I renamed it. Even if it is testing code it should use correct language.

**Pull request merged**